docs/usage.rst

```diff
@@ -253,6 +253,36 @@ For more information, see the
 section in the documentation of the 'zhmcclient' package.
 
 
+Communication with Prometheus
+-----------------------------
+
+The exporter is an HTTP or HTTPS server that is regularly contacted by Prometheus
+for collecting metrics using HTTP GET.
+
+The parameters for the communication with Prometheus are defined in the
+HMC credentials file in the ``prometheus`` section, as in the following example:
+
+.. code-block:: yaml
+
+    prometheus:  # optional
+      port: 9291
+      scheme: https
+      server_cert_file: server_cert.pem
+      server_key_file: server_key.pem
+      ca_cert_file: ca_certs.pem
+      verify_cert: true
```

Comment:
@andy-maier Also, hwo about omitting the "verify_cert" setting as well. Instead, use the logic that enable mTLS to verify client certificate if the "ca_cert_file" is provided together with the "server_cert_file" and "server_key_file" settings. Or, just enable simple TLS not to verify client certificate.

Reply:
DONE. The update removes "verify_cert" and implies it from the presence of "ca_cert_file". This means we can no longer use the default certificates, but they were badly documented anyway.